### ISSUE TITLE
Unhandled error on saving backup for New document

### ISSUE THREAD

**Issue by AdrianoFerrari** [gingko/client#157]
Issue is on main event `doc:save-backup`, if the document is Untitled/New.

```
Unhandled Error
TypeError: path must be a string or Buffer
    at Object.stat (/home/adriano/code/learn/gingko/client/app/electron.js:67693:19)
    at fs.stat (/home/adriano/code/learn/gingko/client/app/electron.js:68457:8)
    at /home/adriano/code/learn/gingko/client/app/electron.js:67697:20
```

**Comment by 8bfcpcch** [comment]
This is all Greek to me, but I may not have given my document a title when I got the error I reported upon closing.

**Comment by AdrianoFerrari** [comment]
Fixed in [v2.4.2](https://github.com/gingko/client/releases/tag/v2.4.2).
